src/cryptojwt/serialize/__init__.py

```diff
@@ -0,0 +1,47 @@
+
+class SimpleList():
```

Comment:
Suggest we add a pure abstract base class (e.g., `SerializeBase`) and let SimpleList be a non-astract version of this. However, I cannot see that `SimpleList` is used anywhere in the code, is it? Leftovers?

Reply:
Leftovers.